src/widgets/span/lv_span.h

```diff
@@ -239,6 +250,40 @@ uint32_t lv_spangroup_get_expand_width(lv_obj_t * obj, uint32_t max_width);
  */
 int32_t lv_spangroup_get_expand_height(lv_obj_t * obj, int32_t width);
 
+/**
+ * Get the span's coords in the spangroup.
+ * @note Before calling this function, please make sure that the layout of span group has been updated.
+ * +--------+
+ * |Heading +--->------------------+
+ * |  Pos   |   |     Heading      |
+ * +--------+---+------------------+
+ * |                               |
+ * |                               |
+ * |                               |
+ * |            Middle   +--------+|
+ * |                     |Trailing||
+ * |                   +-|  Pos   ||
+ * |                   | +--------+|
+ * +-------------------v-----------+
+ * |     Trailing      |
+ * +-------------------+
+ * @param obj       pointer to a spangroup object.
+ * @param span      pointer to a span.
+ * @return the span's coords in the spangroup.
+ */
+lv_span_coords_t lv_spangroup_get_span_coords(lv_obj_t * obj, lv_span_t * span);
```

Comment:
It is not recommended to pass large variables as return values, it should be passed as pointer in arguments.
Passing the `spangroup_t` can be avoided, as each span has a spangroup field.

`void lv_span_get_relative_coords(lv_span_t * span, lv_span_coords_t *)`

Reply:
Don't worry, I think the compiler will optimize the struct return.